From: faqman8 <faqman8@outlook.com>
Date: Sun, 20 Apr 2025 09:13:30 +0000
Subject: Translated using Weblate (Vietnamese)

Currently translated at 100.0% (210 of 210 strings)

Translation: openkylin-nile-new/kylin-os-manager-device-controller
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager-device-controller/vi/
---
 .../kylin-os-manager-device-controller_vi.ts       | 28 +++++++++++-----------
 1 file changed, 14 insertions(+), 14 deletions(-)

diff --git a/plugins/device-controller/translations/kylin-os-manager-device-controller_vi.ts b/plugins/device-controller/translations/kylin-os-manager-device-controller_vi.ts
index 66cc3f5..c15a078 100644
--- a/plugins/device-controller/translations/kylin-os-manager-device-controller_vi.ts
+++ b/plugins/device-controller/translations/kylin-os-manager-device-controller_vi.ts
@@ -837,19 +837,19 @@
         <location filename="../src/include/hwinfostorage.h" line="58"/>
         <location filename="../src/include/hwinfostorage.h" line="71"/>
         <source>Part Number</source>
-        <translation>Phần số</translation>
+        <translation>Chương trình mảng</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="59"/>
         <location filename="../src/include/hwinfostorage.h" line="72"/>
         <source>Memory Usage</source>
-        <translation>Sử dụng bộ nhớ</translation>
+        <translation>Tỷ lệ sử dụng bộ nhớ</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="77"/>
         <location filename="../src/include/hwinfostorage.h" line="95"/>
         <source>Sub System Device ID</source>
-        <translation>ID thiết bị hệ thống phụ</translation>
+        <translation>Số thiết bị hệ thống con</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="78"/>
@@ -871,7 +871,7 @@
         <location filename="../src/include/hwinfostorage.h" line="79"/>
         <location filename="../src/include/hwinfostorage.h" line="97"/>
         <source>Memory</source>
-        <translation>Trí nhớ</translation>
+        <translation>Bộ nhớ</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="80"/>
@@ -887,7 +887,7 @@
         <location filename="../src/include/hwinfostorage.h" line="312"/>
         <location filename="../src/include/hwinfostorage.h" line="321"/>
         <source>Bus Info</source>
-        <translation>Thông tin xe buýt</translation>
+        <translation>Thông tin Bus</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="82"/>
@@ -895,7 +895,7 @@
         <location filename="../src/include/hwinfostorage.h" line="221"/>
         <location filename="../src/include/hwinfostorage.h" line="228"/>
         <source>Clock</source>
-        <translation>Đồng hồ</translation>
+        <translation>Tần số xung nhịp</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="83"/>
@@ -939,7 +939,7 @@
         <location filename="../src/include/hwinfostorage.h" line="86"/>
         <location filename="../src/include/hwinfostorage.h" line="104"/>
         <source>Communication Width</source>
-        <translation>Bề rộng bit truyền thông</translation>
+        <translation>Độ rộng bit truyền thông</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="166"/>
@@ -957,7 +957,7 @@
         <location filename="../src/include/hwinfostorage.h" line="223"/>
         <location filename="../src/include/hwinfostorage.h" line="230"/>
         <source>Width</source>
-        <translation>Bề rộng bit</translation>
+        <translation>Độ rộng bit</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="87"/>
@@ -1023,7 +1023,7 @@
         <location filename="../src/include/hwinfostorage.h" line="136"/>
         <location filename="../src/include/hwinfostorage.h" line="150"/>
         <source>Bus</source>
-        <translation>Thông tin bus</translation>
+        <translation>Thông tin Bus</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="138"/>
@@ -1088,7 +1088,7 @@
         <location filename="../src/include/hwinfostorage.h" line="171"/>
         <location filename="../src/include/hwinfostorage.h" line="184"/>
         <source>Main Disk</source>
-        <translation>Ổ cứng（Có/Không）</translation>
+        <translation>Ổ cứng chính （Có/Không）</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="173"/>
@@ -1112,7 +1112,7 @@
         <location filename="../src/include/hwinfostorage.h" line="197"/>
         <location filename="../src/include/hwinfostorage.h" line="209"/>
         <source>Monitor Ratio</source>
-        <translation>Hiện thị tỷ lệ khung hình</translation>
+        <translation>Tỷ lệ khung hình hiển thị</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="198"/>
@@ -1163,7 +1163,7 @@
         <location filename="../src/include/hwinfostorage.h" line="220"/>
         <location filename="../src/include/hwinfostorage.h" line="227"/>
         <source>Bus Address</source>
-        <translation>Địa chỉ bus</translation>
+        <translation>Địa chỉ Bus</translation>
     </message>
     <message>
         <location filename="../src/include/hwinfostorage.h" line="261"/>
@@ -1283,7 +1283,7 @@
     <message>
         <location filename="../src/hardwareinfo/graphiccardinfo.cpp" line="109"/>
         <source>This graphics card does not support obtaining GPU bus bit width</source>
-        <translation>Dòng card đồ họa này không hỗ trợ chức năng hiển thị bộ nhớ đệm rộng</translation>
+        <translation>Dòng card đồ họa này không hỗ trợ chức năng hiển thị độ rộng bit</translation>
     </message>
     <message>
         <location filename="../src/driveroperation/uninstaller.cpp" line="51"/>
@@ -1293,7 +1293,7 @@
     <message>
         <location filename="../src/driveroperation/aptinstaller.cpp" line="62"/>
         <source>Install timeout.</source>
-        <translation>Lỗi quá giờ cài đặt.</translation>
+        <translation>Lỗi quá thời gian cài đặt.</translation>
     </message>
 </context>
 <context>
